che-terminal-client/package.json

```diff
@@ -27,15 +27,16 @@
     "src/**/*.ts"
   ],
   "devDependencies": {
+    "2": "1.0.2",
```

Comment:
what is that new dependency ?

Reply:
no idea, it was pulled in via the frontend maven plugin when running the build. If you run the build you should also see that change. :D